### ISSUE TITLE
Generate hash from command-line

### ISSUE THREAD

**Issue by CaptainBrad** [digital-preservation/droid#849]
Bug pointed out by a user - 

Hi

Is there a way to generate the hash sha256 using `droid` from the command line?

Interestingly, in the source code[1], there is an example:

```
droid -Pr "profile.maxBytesToScan=32768" -a "C:\\Files\\A Folder" -p "C:\\Results\\result1.droid".\n
```

But when I try:

```
PS C:\Users\garcm0b\Work\droid-binary-6.5.2-bin-win32-with-jre> .\droid.bat -Pr "profile.generateHash=true" -a "C:\Users\garcm0b\Work\013_002_0026" -p "C:\Users\garcm0b\Work\013_002_0026\013_002_0026.droid"
(...)
Incorrect command line syntax: org.apache.commons.cli.UnrecognizedOptionException: Unrecognized option: -Pr
```

In my case, `droid` don't recognize the `-Pr` option, but I don't know why.

So, how to generate hash from command line?

[1]
https://github.com/digital-preservation/droid/blob/master/droid-command-line/src/main/resources/options.properties

**Comment by steve-daly** [comment]
I've worked out what's going on here.  The syntax above is trying to add command line parameters to the DROID GUI not to the DROID command line utility.  

I tried the syntax on the command line version and it works fine, so there is no problem.

**Comment by Dclipsham** [comment]
I'm not so sure. The user is executing droid.bat with params given, which is what routes the .bat file to launch the CLI version rather than the GUI. Without params droid.bat will launch the GUI.

I believe the issue is that, judging by the path, the user is launching 6.5.2, but I don't believe the -Pr flag was added until 6.6.0 as per https://github.com/digital-preservation/droid/commit/1698359f80388b0aadfd028fb04e1aa5580e3a04

**Comment by steve-daly** [comment]
Yes, @Dclipsham is right.  I'll reopen.

**Comment by steve-daly** [comment]
I believe the comment from @Dclipsham is the simple cause of this issue.  

The notes in the original report are taken from the source code of unreleased feature.  The command above works in the forthcoming 6.6.0 but not in the released 6.5.2.
